packages/fungi/fungi.0.1.3/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "A pure functional graph library"
+description: """\
+A graph library based on functional adjacency list with some common graph
+algorithms implemented and a fibonacci heap."""
+maintainer: "Harry K <kituyiharry@github.com>"
+authors: "Harry K <harryk@harryk.dev>"
+license: "MIT"
+tags: ["topics" "graph" "functional" "networks"]
+homepage: "https://harryk.dev/articles/article1"
+bug-reports: "https://github.com/kituyiharry/fungi/issues"
+depends: [
+  "ocaml" {>= "4.14.0"}
+  "dune" {>= "3.12" & >= "3.12"}
```

Comment:
There's a redundant version specifier. I think caused by this line: https://github.com/kituyiharry/fungi/blob/eac5d9007db1306c6ea89c1658eb2aa633691edf/dune-project#L39

You actually don't need to explicitly specify that the project depends on dune, dune will insert that into the opam file automatically.